### ISSUE TITLE
train loss is to high when i run train.py in train/voc-fcn.is it normal?

### ISSUE THREAD

**Issue by zz10001** [ZijunDeng/pytorch-semantic-segmentation#51]
![image](https://user-images.githubusercontent.com/39954774/59238953-79ba0b80-8c32-11e9-8d4c-ce7578456b2f.png)

**Comment by zhangjiyuzjy** [comment]
modify loss function parameter

**Comment by zz10001** [comment]
can you tell the detail,i have try many times,but it don't works.

**Comment by zz10001** [comment]
> modify loss function parameter

can you tell the detail,i have try many times,but it don't works.

**Comment by zhangjiyuzjy** [comment]
size_average  set to True

**Comment by zz10001** [comment]
thx,i will try it

**Comment by zhangjiyuzjy** [comment]
do you test other models

**Comment by zhangjiyuzjy** [comment]
if you test and get good result ,can you give me some help

**Comment by zz10001** [comment]
> if you test and get good result ,can you give me some help
sorry,i haven't try others model,because i just download the voc datasets.i just run voc-fcn.and the loss problem.i didn't run others.

**Comment by zhangjiyuzjy** [comment]
this mode can run but the prediction result is poor,maybe is my fault,if you get good result,please connect me

**Comment by zhangjiyuzjy** [comment]
i train this model loss is 0.07,but prediction is so poor

**Comment by zz10001** [comment]
> this mode can run but the prediction result is poor,maybe is my fault,if you get good result,please connect me
it seem works when i modify size_average set to True.but the loss is still...like this.is it better after 100 epoch, if i get better result i will tell you!
![image](https://user-images.githubusercontent.com/39954774/59591203-2e1acc80-9120-11e9-827b-f1e599b8922f.png)

**Comment by zhangjiyuzjy** [comment]
you use benchmark dataset,this need more time,thank you share go on to run
